File: Templates/Examples/NHibernate/VisualBasic/NHibVbSample.Data/ManagerObjects/InventoryManager.vb
Imports System
Imports System.Collections.Generic
Imports System.Text

Imports NHibernate
Imports NHibVbSample.Generated.BusinessObjects
Imports NHibVbSample.Generated.Base

Namespace NHibVbSample.Generated.ManagerObjects
	'Public Partial Interface IInventoryManager
	'	Inherits IManagerBase(Of Inventory, System.String)
	'End Interface

	Partial Class InventoryManager
		Inherits ManagerBase(Of Inventory, System.String)
		Implements IInventoryManager
	End Class
End Namespace

File: Projects/Framework-Samples/NHibernate/CSharp/NHibCsSample.Data/ManagerObjects/InventoryManager.cs
using System;
using System.Collections.Generic;
using System.Text;

using NHibernate;
using NHibCsSample.Generated.BusinessObjects;
using NHibCsSample.Generated.Base;

namespace NHibCsSample.Generated.ManagerObjects
{
    public partial interface IInventoryManager : IManagerBase<Inventory, string>
    {
	}
	
	partial class InventoryManager : ManagerBase<Inventory, string>, IInventoryManager
    {
	}
}